heudiconv/info.py

```diff
@@ -43,7 +43,10 @@
     'extras': [
         'duecredit',  # optional dependency
     ],  # Requires patched version ATM ['dcmstack'],
-    'datalad': ['datalad >=%s' % MIN_DATALAD_VERSION]
+    'datalad': ['datalad >=%s' % MIN_DATALAD_VERSION],
+    'physio': [
+        'bidsphysio.dcm2bids >=1.4.3; python_version>"3.5"',   # if dicoms with physio need to be converted
+    ]
 }
 
 # Flatten the lists
```

Comment:
all those binary files below adding up to over 1MB -- I think should be moved to some external dataset and used via `fetch_data`. ATM `fetch_data` hardcodes to access from datasets.datalad.org but I think we should change that -- if full url is provided - just use that URL, and then share them on some repo, could be even not annex, just regular git repo on github.

Later we should improve this `fetch_data` to make data persistent locally etc... but not for this PR.